### DISCUSSION TITLE
Is incremental record linking/deduplication supported?

### DISCUSSION THREAD

**Discussion by lmores** [moj-analytical-services/splink#822] [Q&A] [answered]
I have found this really interesting library and reading the documentation I haven't found out whether it is possible to incrementally update the result of record deduplication analysis when new records are added to a table that has already been analyzed with splink.
So, if new records are added to a table that has already been scanned, is there a way to execute record deduplication without recomputing everything from scratch?

**Comment by RobinL** [comment, marked as answer]
Yes - this is supported to some extent.  

See here: https://moj-analytical-services.github.io/splink/demos/example_real_time_record_linkage.html

The specific function you want is probably:
https://moj-analytical-services.github.io/splink/linker.html#splink.linker.Linker.find_matches_to_new_records

This will be a little easier to work with once the following PR is merged in the next few days:
https://github.com/moj-analytical-services/splink/pull/811
